README.md

```diff
@@ -24,7 +24,7 @@ Try
 
 `docker --version`
 
-If there is no version installed, follow the instructions to install it
+If there is no version installed, follow the instructions to install it [here](https://www.docker.com/products/personal/)
```

Comment:
Also, super minor, but the instructions differ by operating system, so maybe use this link instead? https://docs.docker.com/engine/install/
My bad for not sending you that one in the first place!

Reply:
Ah good call. Fixed